i18n-config/src/main/resources/META-INF/i18n/fr_fr.conf

```diff
@@ -0,0 +1,111 @@
+  "   §1  / /| |§9/ __§3/ __§6/ §e/ /§6 _ / _ \\/__/"
+  "   §1 / ___ §9/ / §3/ /_§6/ §e/ /§6 / / // / /"
+  "   §1/_/  |§9/_/  §3\\__§6/_§e/_/§6\\_ /_//_/ /"
+  "   §1      §9     §3    §6  §e §6/__/     \\/"
+  ""
+  "    §aVersion {}"
+  "    §aDate du Build {}"
+  ""
+]
+downloader {
+  info2 = [
+    "Librairies manquantes, téléchargement en cours !"
+    "Soutenir le projet MinecraftForge https://www.patreon.com/LexManos/"
+  ]
+  http-error = "Erreur HTTP {0} {1}"
+  not-found = "Non trouvé dans {0}"
+  redirect-error = "Erreur de redirection {0}"
+  dir = "Dossier existant dans la destination {0}"
+  timeout = "Timeout {0} {1}"
+  hash-not-match = "SHA1 du fichier non-correspondant, SHA1 attendu -> {0} SHA1 trouvé -> {1}: {2}"
+  maven-fail = "{0} échec du téléchargement: {1}"
+  complete = "{0} téléchargement terminé"
+  forge-install = "L''installation de Forge démarre, veuillez patienter... "
+  access-denied = "Accès d'écriture du fichier refusé {0}: {1}"
+}
+
+implementer {
+  not-found = "Class non trouvée {}"
+  error = "Erreur survenue {}"
+}
+i18n {
+  current-not-available = "Les paramètres de langue {0} ne sont pas disponibles"
+  using-language = "Utilisation des paramètres de langue {0} et des paramètres de langue de secours {1}"
+}
+loading-mapping = "Chargement des mappings ..."
+mixin-load {
+  core = "Arclight core mixin ajouté."
+  optimization = "Arclight optimization mixin ajouté."
+}
+mod-load = "Arclight Mod chargé."
+patcher {
+  loading = "Chargement des patchs de plugins ..."
+  loaded = "{} patchs chargés"
+  load-error = "Une erreur s'est produite lors du chargement d'un patch"
+}
+registry {
+  forge-event = "Événements Arclight enregistrés."
+  begin = "Enregistrement de Bukkit ..."
+  error = "Une erreur s'est produite lors de l'enregistrement de Forge "
+  enchantment = "{} enchantements enregistrés"
+  potion = "{} nouveaux effets de potions enregistrés"
+  material = "{} nouveaux materiaux enregistrés dont {} blocs et {} objets"
+  entity-type = "{} nouveaux types d'entités enregistrés"
+  environment = "{} nouvelles dimensions enregistrés"
+  villager-profession = "{} nouvelles professions de villageois enregistrés"
+  biome = "{} nouveaux biomes enregistrés"
+  meta-type {
+    not-subclass = "{} n'est pas une sous-classe de {}"
+    error = "{} itemMetaType {} fourni n'est pas valide: {}"
+    no-candidate = "{} aucun constructeur valide trouvé pour itemMetaType {}"
+  }
+  block-state {
+    not-subclass = "{} n'est pas une sous-classe de {}"
+    error = "{} itemMetaType {} fourni n'est pas valide: {}"
+    no-candidate = "{} aucun constructeur valide trouvé pour blockStateClass {}"
+  }
+  entity {
+    not-subclass = "{} n'est pas une sous-classe de {}"
+    error = "{} entityClass {} fourni n'est pas valide: {}"
+  }
+}
+dfu-disable {
+  legacy-plugin = "Les plugins "Legacy" ne sont pas autorisés lorsque optimization.disable-data-fixer est activé !"

```

Comment:
I guess there're some misunderstanding here. Legacy plugin means plugins that write for versions before 1.13 but not a plugin named "Legacy". And if you want to have quote inside quote you need add a backslash before them (`" \" "`).